src/components/collapsible-section/collapsible-section.scss

```diff
@@ -108,10 +107,10 @@ header {
 .divider-line {
     transition: opacity 0.3s ease 0.3s;
     flex-grow: 1;
-    height: functions.pxToRem(2);
-    border-radius: functions.pxToRem(1);
+    height: 0.125rem;
+    border-radius: 1rem;
```

Comment:
I know this number was equivalent to `1px`. But the generated result will be the same as `16px`. For simplicity I wrote `1rem`, instead of a weird number.

Reply:
I just meant for a component that is 2px high, a border radius of 1px or 9999999px will generate the exact visual result. So for simplicity, I didn't write `0.0625rem`. I just wrote `1rem`. It's shorter. :)

Reply:
Ah! Haha! Of course! I didn't consider the height. It's been a while since I worked much in CSS, so I guess I'm a little rusty 😆

Yeah, perfect 👍